libsrc/vic20/tgi/vic20-hi.s

```diff
@@ -0,0 +1,1054 @@
+;
+; Graphics driver for a 160x192x2 mode on the VIC-20.
+;
+; Based on C64 TGI driver
+;
+; 2018-03-11, Sven Michael Klose
+;
+
+        .include        "zeropage.inc"
+
+        .include        "tgi-kernel.inc"
+        .include        "tgi-error.inc"
+
+        .macpack        generic
+        .macpack        module
+
+
+; ------------------------------------------------------------------------
+; Constants
+
+COLS := 20
```

Comment:
Numeric constants are supposed to be set via `=`. See https://cc65.github.io/doc/ca65.html#ss6.1